### PR TITLE
Refactor prefer rules

### PR DESCRIPTION
**Pull request by xfumihiro**
fixes #20

### CODE REVIEW

**Comment by xfumihiro** [comment]
btw, do we need this? [Commit Message Guidelines](https://gist.github.com/abravalheri/34aeb7b18d61392251a2)

**Comment by SimenB** [comment]
I'm all for adding conventional commits, and validating them

File: rules/prefer_to_have_length.js [hunk 1]
```diff
@@ -1,46 +1,44 @@
 'use strict';
+const {
```
Comment (SimenB) [inline rules/prefer_to_have_length.js]: Destructure doesn't work in node 4. We can revisit dropping node 4 later, but for now can you make this work there?

**Comment by xfumihiro** [comment]
Should we use [this](https://github.com/conventional-changelog/standard-version) then?

**Comment by SimenB** [comment]
I'm thinking of setting up https://github.com/semantic-release/semantic-release/.

**Comment by SimenB** [comment]
Thanks for this! I'll try to set up automatic releases later today or tomorrow.

For now, released as 21.4.0

**Pull request merged**